DTXMania/Code/Stage/04.Config/CActConfigList.cs

```diff
@@ -713,11 +700,11 @@ public void tSetupItemList_Drums()
                 "とめた表示です。\n" +
                 "Type-C 3タムのみをまとめた表示です。\n" +
                 "Type-D 左右完全対象の表示です。",
-                "To change the displaying position of\n" +
+                "Change the display position of\n" +
                 "Drum Lanes.\n" +
-                "Type-A default\n" +
-                "Type-B Summarized 2 pedals and Toms.\n" +
-                "Type-C Summarized 3 Toms only.\n" +
+                "Type-A: Default\n" +
+                "Type-B Summarized two pedals and Toms.\n" +
```

Comment:
**question: trivial** I'm not sure if this description fully explains this setting. The Japanese translation wasn't that clear.

Reply:
This option visually swaps the position of lanes - will provide a better translation for this.